### ISSUE TITLE
Punctiform multicolored opacities in the posterior corneal stroma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7575]
**Preferred term label:**
Punctiform multicolored opacities in the posterior corneal stroma

**Synonyms**

**Definition (free text, please give PubMed ID)**
the presence of thin, punctiform elements of various colors in the posterior stroma of the cornea, immediately anterior to Descemet membrane. The elements are visible to direct and indirect illumination.
PMID:29728753

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID:31782998
12/12

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
